src/eh1/delay.rs

```diff
@@ -74,11 +74,9 @@ pub struct Transaction {
 }
 
 /// Nanoseconds per microsecond
-const NANOS_PER_MICRO: u32 = 1_000;
+const NANOS_PER_MICRO: u64 = 1_000;
 /// Nanoseconds per millisecond
-const NANOS_PER_MILLI: u32 = 1_000_000;
-/// Microseconds per millisecond
-const MICROS_PER_MILLI: u32 = 1_000;
+const NANOS_PER_MILLI: u64 = 1_000_000;
```

Comment:
Nit: I'd personally prefer `*_PER_US` and `*_PER_MS` to be aligned with the `*_delay_us` and `*_delay_ms` methods. But if you prefer the variant above, feel free to keep it.